### ISSUE TITLE
Tasks are reporting statusCode instead of status

### ISSUE THREAD

**Issue by evasomething** [apache/incubator-druid#6333]
In the task status payload, tasks are returning `statusCode` instead of `status`. I thought this was expected new behavior but have been told by @gianm I'm mistaken and this is a bug.

sample response from `/runningTasks` endpoint
```
    {
      "id": "index_kinesis_kinesis-test_007c7ecb6b7a79c_aogcbmlh",
      "type": "index_kinesis",
      "createdTime": "2018-08-16T01:55:03.207Z",
      "queueInsertionTime": "2018-08-16T01:55:03.210Z",
      "statusCode": null,
      "duration": null,
      "location": {
        "host": "10.0.0.4",
        "port": 8100,
        "tlsPort": -1
      },
      "dataSource": "kinesis-test",
      "errorMsg": null
    }
```

sample response from `/{taskId}/status` endpoint
```
{
  "task": "index_shutdown_fe_test",
  "status": {
    "id": "index_shutdown_fe_test",
    "type": "index",
    "createdTime": "2018-06-29T19:31:37.609Z",
    "queueInsertionTime": "1970-01-01T00:00:00.000Z",
    "statusCode": "FAILED",
    "runnerStatusCode": "WAITING",
    "duration": -1,
    "location": {
      "host": null,
      "port": -1,
      "tlsPort": -1
    },
    "dataSource": "dataset_fe_test",
    "errorMsg": null
  }
}
```

**Comment by gianm** [comment]
They return "status" in 0.12.x so I think this is new in master.

**Comment by surekhasaharan** [comment]
Looking at the history of this file `TaskStatusPlus`, it seems it was `state` before being changed to `statusCode` https://github.com/apache/incubator-druid/pull/5469/files#diff-8bc272789f1858fad01af76a1052506dR46. So do we change it back to `state` or `status` ?

**Comment by fjy** [comment]
Fixed by #6334
